android/capacitor/src/main/java/com/getcapacitor/plugin/util/HttpRequestHandler.java

```diff
@@ -398,6 +398,19 @@ public static JSObject request(PluginCall call, String httpMethod, Bridge bridge
 
         boolean isHttpMutate = method.equals("DELETE") || method.equals("PATCH") || method.equals("POST") || method.equals("PUT");
 
+        // a workaround for the following android web view issue:
+        // https://issues.chromium.org/issues/40450316
+        // x-cap-user-agent contains the user agent set in JavaScript
+        String userAgentValue = headers.getString("x-cap-user-agent");
+        if (userAgentValue != null && !userAgentValue.isEmpty()) {
```

Comment:
```suggestion
        if (userAgentValue != null) {
```
I noticed iOS allows empty strings, so we should also allow them for Android